### PR TITLE
Update superset deployment image

### PR DESCRIPTION
**Pull request by 4n4nd**
This will update the Superset Deployment image to `quay.io/opendatahub/superset:1.1.0`, which has the drivers for superset to be able to connect with Trino as a db.

### CODE REVIEW

**Comment by LaVLaS** [comment]
@maulikjs Is there any reason why `superset:1.1.0-trino` should not be the default superset image?

**Comment by maulikjs** [comment]
@LaVLaS no reason, we actually just pushed this image recently and then added trino so this image exists, 1.2.0 onwards `trino` will be part of the default image.

File: superset/base/params.env [hunk 1]
```diff
@@ -16,4 +16,4 @@ SUPERSET_DB_MEMORY_REQUESTS=300Mi
 SUPERSET_DB_MEMORY_LIMITS=1Gi
 SUPERSET_DB_CPU_REQUESTS=300m
 SUPERSET_DB_CPU_LIMITS=1
-SUPERSET_IMAGE=quay.io/internaldatahub/superset:1.1.0
+SUPERSET_IMAGE=quay.io/internaldatahub/superset:1.1.0-trino
```
Comment (anishasthana) [inline superset/base/params.env:19]: Should this image reside in the opendatahub org on Quay?

Comment (4n4nd) [reply superset/base/params.env:19]: @maulikjs @LaVLaS

Comment (rimolive) [reply superset/base/params.env:19]: `opendatahub/superset` is empty, so I decided to keep the version tag without the additional name: 

```
quay.io/opendatahub/superset:1.1.0
```

@4n4nd change the code to point to this image

Comment (4n4nd) [reply superset/base/params.env:19]: does it have the trino driver?

Comment (rimolive) [reply superset/base/params.env:19]: Yes, I pulled from the `1.1.0-trino` in internaldatahub org to there. I just didn't use the `-trino` suffix.

Comment (4n4nd) [reply superset/base/params.env]: updated :+1:

**Comment by anishasthana** [comment]
It's deployed on the OPF cluster, changes look good.

**Comment by 4n4nd** [comment]
@rimolive is `quay.io/internaldatahub/superset:1.1.0-trino` the same as `quay.io/opendatahub/superset:1.1.0`? If so, we have `quay.io/internaldatahub/superset:1.1.0-trino` deployed in OperateFirst.

**Review by maulikjs — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]

**Pull request merged**